### PR TITLE
Added community best practices

### PR DESCRIPTION
**Pull request by kkiesling**
These are the files necessary for satisfying the community best practices checklist [here](https://github.com/svalinn/DAGMC/community) and satisfy issue #587 and somewhat satisfy #567.

Please provide comments on parts you want to see changed or if there are missing sections in any of the guidelines.

My own news file for this PR incoming when I know the PR number.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Contributing to DAGMC
+=====================
+
+The following is a guide to contributing to the DAGMC-Viz repository.
```
Comment (gonuke) [inline CONTRIBUTING.md]: "DAGMC-Viz" -> "DAGMC"  (maybe a quick search of all files for this?)

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks @kkiesling  - this will definitely help me follow all the rules!

I just found one issue where it still says "DAGMC-Viz"

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+Contributor Code of Conduct
+===========================
+
+Our Pledge
+----------
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers of CNERG pledge to making participation in our project and
```
Comment (kkiesling) [inline CODE_OF_CONDUCT.md]: @gonuke - Do you think this should say the "maintainers of CNERG" or "maintainers of Svalinn"?

File: CONTRIBUTING.md (hunk 1 above)
Comment (kkiesling) [reply CONTRIBUTING.md]: Whoops. Fixed that.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (gonuke) [reply CODE_OF_CONDUCT.md]: maybe "the Svalinn software suite, including DAGMC,"

**Review by gonuke — APPROVED** [review]

**Comment by kkiesling** [comment]
Updated the organization name in the code of conduct. This should be good to go now.

**Comment by gonuke** [comment]
Thanks @kkiesling

**Pull request merged**